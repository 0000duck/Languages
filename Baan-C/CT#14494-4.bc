|This is a dll script, which does not have any sections.
function extern long func1(long var1)
{
    |do something
}

function void funct2(long var2)
{
    |do something
}