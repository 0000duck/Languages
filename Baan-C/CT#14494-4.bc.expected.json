{
    "root":"CT#14494-4.bc",
    "leaves":[
        "func1",
        "funct2"
    ]
)